src/particle/particle/particle.py

```diff
@@ -564,7 +570,7 @@ def load_table(cls, filename=None, append=False, _name=None):
                 cls.load_table(f, append=True, _name="nuclei2020.csv")
             return
         elif isinstance(filename, HasRead):
-            tmp_name = _name or getattr(filename, "name")
+            tmp_name = _name or filename.name
```

Comment:
Not any safer than the simple expression. Unless we intended to have it "fall-back" on an empty string or something (in which chase it needs the three argument form).

Reply:
No. Your update seems fine.